db/model/Variable.ts

```diff
@@ -371,3 +402,98 @@ export const getOwidVariableDisplayConfig = async (
     if (!row.display) return
     return JSON.parse(row.display)
 }
+
+const readValuesFromMysql = async (
+    variableId: OwidVariableId
+): Promise<any[]> => {
```

Comment:
Is it okay having the `any` here? Will we use this in places where we'd like it to be typed?

Reply:
It wasn't there before, but we should definitely have that! Thanks for spotting this.